packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -603,6 +620,10 @@ export async function downloadSnapshot(
 			}
 		});
 	}
+	if (groupIds.length > 0) {
+		// eslint-disable-next-line @typescript-eslint/dot-notation
+		queryParams["groupId"] = Array.from(groupIds).join(",");
```

Comment:
how do we tell SPO that we want "default" group? We should differentiate from legacy clients who can't work with partial snapshots, and thus are asking for whole snapshot?

Reply:
The new flow is passing "default" groupid while the old flow is not passing any groupid. So, old clients will not use getSnapshot api and will use getVersions flow. Only the new flow will pass the groupids. BY default, if no groupid is asked for then "default" groupid will be asked for.

Reply:
You can follow the code from odspDocumentStorageManager class, that would make it clearer.

Reply:
Somewhat related (after looking further) - groupIds.length is always positive here (with current design), due to using of defaultGroupIdForSnapshot in getSnapshot(). I suggested to change that and use defaults here instead.